tests/integration/steps/integration_steps.go

```diff
@@ -0,0 +1,699 @@
+package steps
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	resty "github.com/go-resty/resty/v2"
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var namespace = "k8ssandra"
+var repairId string
+
+func RunShellCommand(command *exec.Cmd) error {
+	err := command.Start()
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = command.Wait()
+	return err
+}
+
+func RunShellCommandAndGetOutput(command *exec.Cmd) string {
+	var outb bytes.Buffer
+	command.Stdout = &outb
+	err := command.Run()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return string(outb.String())
+}
+
+func GetServicesWithLabel(label string) (*v1.ServiceList, error) {
```

Comment:
Go does not use "Get" as prefix.

Reply:
I propose we create a basic `Go standards guide` to be on the same page.  We have a few folks that are learning Go and have background with numerous other languages where it is acceptable to have getters/setters.

Reply:
https://golang.org/doc/effective_go

Reply:
It's not really a getter since username/password are not class members (or anything equivalent in Go). I'll rename to "Extract..." since it's more accurate with what is being done.